puffin/Cargo.toml

```diff
@@ -28,9 +29,12 @@ serde = { version = "1.0", features = ["derive", "rc"], optional = true }
 [target.'cfg(not(target_arch = "wasm32"))'.dependencies]
 zstd = { version = "0.11.1", optional = true } # native only
 
+# web:
 [target.'cfg(target_arch = "wasm32")'.dependencies]
+js-sys = { version = "0.3" }
```

Comment:
Hum… is depending on `instant` fine as long as it is not being called?

Reply:
think can be fine, though does bring in unnecessary dependency with bunch of javascript bindings which increases build times. so optional is better.

calling it will panic

Reply:
Done! I've added a "web" feature (opt-in). Without it `puffin` still depend on "instant", but [that doesn't bring inany other dependencies than `cfg-if`](https://github.com/sebcrozet/instant/blob/master/Cargo.toml)